src/libsync/discoveryphase.h

```diff
@@ -142,13 +143,18 @@ class DiscoverySingleDirectoryJob : public QObject
 {
     Q_OBJECT
 public:
-    explicit DiscoverySingleDirectoryJob(const AccountPtr &account, const QString &path, QObject *parent = nullptr);
+    explicit DiscoverySingleDirectoryJob(const AccountPtr &account,
+                                         const QString &path,
+                                         const QSet<QString> &topLevelE2eeFolderPaths,
```

Comment:
@mgallien we may want to improve that, I am pretty sure I had to do it for some reason, but now, sorry sorry, we just don't have time for that, tomorrow I am off and I don't want to spend whole night fixing issues because of changing this, I will mark it as TODO and will fix first thing when I am available again

Reply:
at least now, it works and been tested and lots of issues were fixed, if I change this now, this means many extra hours to fix everything back